src/App.tsx

```diff
@@ -1,61 +1,97 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { ToDoWithUser } from './components/types';
 
 export const App = () => {
+  const todosWithUsers = todosFromServer.map(todo => {
+    const user = usersFromServer[usersFromServer
+      .map(userFromServer => userFromServer.id)
+      .indexOf(todo.userId)];
+
+    return { ...todo, user };
+  });
```

Comment:
you can write a function for it and move it outside component file ( recommendation)